# 現在の運転状態SET
uint32 robotStat
---

